### PR TITLE
[cker] Introduce CategoricalCrossEntropyWithLogits

### PR DESCRIPTION
**Pull request by ragmani**
This commit introduces CategoricalCrossEntropyWithLogits that computes cce loss value and gradient with nomalization(softmax).

ONE-DCO-1.0-Signed-off-by: ragmani <ragmani0216@gmail.com>

### CODE REVIEW

**Comment by ragmani** [comment]
- For #13736
- Draft #13934

**Review by jyoungyun — APPROVED** [review]
LGTM

File: compute/cker/include/cker/eigen/xent_op.h [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+#include "unsupported/Eigen/CXX11/Tensor"
+#include "cker/operation/Helper/Tensor.h"
+
+// From tensorflow/core/kernels/xent_op.h
+namespace nnfw
+{
+namespace cker
+{
+namespace xent_ops
```
Comment (zetwhite) [inline compute/cker/include/cker/eigen/xent_op.h:32]: What is the meaning of `xent`..?

Comment (ragmani) [reply compute/cker/include/cker/eigen/xent_op.h:32]: I'm not sure what `xent` is. Maybe NT-Xent(Normalized Temperature-scaled Cross Entropy Loss)?

Comment (jyoungyun) [reply compute/cker/include/cker/eigen/xent_op.h:32]: In mathematical or graphic representations, `cross` is equated with the variable `x`. So I think Xent means Cross entropy. :)

ref: https://en.wikipedia.org/wiki/X_mark

File: compute/cker/include/cker/eigen/xent_op.h [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+            // longer be trusted
+            this_backprop[j] = this_logits[j] - max_logits;
+            sum = sum + exp(this_backprop[j]);
+          }
+
+          // loss calculation
+          T log_sum = log(sum);
+          for (int j = 0; j < row_size; j++)
+          {
+            loss_sum += this_labels[j] * (log_sum - this_backprop[j]);
```
Comment (zetwhite) [inline compute/cker/include/cker/eigen/xent_op.h:126]: Maybe..? 

```suggestion
            loss_sum += this_labels[j] * (log_sum - log(this_backprop[j]));
```

Comment (zetwhite) [reply compute/cker/include/cker/eigen/xent_op.h:126]: Ah, That was my misunderstanding.

**Review by zetwhite — APPROVED** [review]
LGTM

**Review by zetwhite — DISMISSED** [review]
LGTM

**Comment by hseok-oh** [comment]
@ragmani Please resolve conflict

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by zetwhite — APPROVED** [review]
